study-builder/studies/atcp/contacting-physician.conf

```diff
@@ -0,0 +1,73 @@
+{
+  include required("../snippets/activity-general-form.conf"),
+  "studyGuid": "CMI-AT",
+  "activityCode": "CONTACTING_PHYSICIAN",
+  "versionTag": "v1",
+  "displayOrder": 1,
+  "writeOnce": true,
```

Comment:
This means they can only submit this `contact_physician` once, and cannot edit their physicians afterwards. Is that the goal?

Reply:
yes, for now